### ISSUE TITLE
Incremental resize (instead of using numbers)

### ISSUE THREAD

**Issue by wvlia5** [SublimeText/Origami#141]
I would like that panes could be resized be pressing hotkeys instead of entering numbers

for example, in current pane,
["command+k", "-+rightarrow"] would decrease the size of the current pane by moving the right edge a fixed amount.
["command+k", "++uparrow"] would increase the size of the current pane by moving the top edge a fixed amount.

**Comment by adzenith** [comment]
I don't think you can use `-` and `+` as modifier keys - you can only use the normal modifier keys. I also think it would be difficult to use this multiple times in a row; i.e. you would have to press `["command+k", "++uparrow"]`, `["command+k", "++uparrow"]`, etc., rather than `["command+k", "++uparrow"]`, `["++uparrow"]`, etc. (this is a limitation of Sublime Text).
I'm just not sure what a good set of shortcuts for this would be.

**Comment by wvlia5** [comment]
Gnome does it the following way for resizing windows (i don't know if it's possible in sublime)

Hotkey1 : enters window resizing mode
 Then any number of times:
   Arrows: select one of the four edges
   +- : expand or shrink that edge
Esc: exit window resizing mode

Also I like the idea of having a shortcut for entering into "origami mode", to avoid having to press Command+k many times for doing several origami actions on arrow. Vintage mode already implements stateful/modal functionality
